contents/handbook/people/benefits.md

```diff
@@ -50,13 +50,13 @@ We currently offer a [Training budget](/handbook/people/training#training-budget
 
 As we are fully remote, we provide [all equipment](/handbook/people/spending-money#equipment) you need to have an ergonomic setup at home to be as productive as possible. We provide all team members with a company card for this purpose.
 
-If you ever need change of scenery, we offer $250/month budget towards [coworking or café working](/handbook/people/spending-money#work-space), or [WeWork All Access](https://www.wework.com/solutions/wework-all-access).
+If you ever need change of scenery, we offer a monthly budget towards [coworking or café working](/handbook/people/spending-money#work-space), or [WeWork All Access](https://www.wework.com/solutions/wework-all-access).
 
-> Please message Grace or Tim to get added to our company WeWork account. 
+> Please message Tim to get added to our company WeWork account. 
```

Comment:
this feels like it should be Kendal's responsibility @charlescook-ph ?

Reply:
Changed (Kendal didn't have a WeWork account but have sorted this)